clients/src/main/java/org/apache/kafka/common/requests/UpdateMetadataRequest.java

```diff
@@ -97,6 +102,9 @@ public EndPoint(String host, int port) {
     private static final String PORT_KEY_NAME = "port";
     private static final String SECURITY_PROTOCOL_TYPE_KEY_NAME = "security_protocol_type";
 
+    // Rack
+    private static final String RACK = "rack";
```

Comment:
Also, the comment doesn't add much. Maybe you can just relocate under "EndPoint key name"?